.travis-ci.sh

```diff
@@ -1,4 +1,5 @@
 eval `opam config env`
+sudo apt update
```

Comment:
normally the `depext -u` below should run `apt update`:

```
$ opam depext --help

    -u, --update
           Update the OS package sets before installation

```
Do you have some logs where this is broken?